certified-connectors/PartnerLinq/apiDefinition.swagger.json

```diff
@@ -0,0 +1,190 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PartnerLinq",
+    "description": "PartnerLinq is an integration tool used to connect two or more systems together. Use this connector to connect with PartnerLinq.",
+    "version": "1.0",
+	"contact":{
+		"name":"PartnerLinq Support",
```

Comment:
@FarhanJamilVSI all files should be properly formatted. Please fix the indentation for all files.